src/renderer/index.js

```diff
@@ -970,6 +970,103 @@ var Renderer = Base.extend('Renderer', {
         }
     },
 
+
+    /**
+     * @summary Added from VC-6892. This is a simplified version of _paintCell by removing some unnecessary property assignments
+     * @param {CanvasRenderingContext2D} gc
+     * @param {CellEvent} cellEvent
+     * @param {string} [prefillColor] If omitted, this is a partial renderer; all other renderers must provide this.
+     * @returns {number} Preferred width of renndered cell.
+     * @private
+     * @memberOf Renderer
+     */
+     _simplifiedPaintCell: function(gc, cellEvent, prefillColor) {
```

Comment:
The original function is _paintCell.
I removed the support for 
- native tree column
- row count column
- multiple cell render
The above features are not used anywhere.